examples/v1beta1/pipelineruns/pipelinerun-using-different-subpaths-of-workspace.yaml

```diff
@@ -46,6 +46,8 @@ spec:
           workspace: ws
           subPath: dir-1
     - name: writer-2
+      runAfter:
+        - writer-1
```

Comment:
Interesting! This example was working already I think?  So here we are just kinda documenting the ordering which the PVC accessMode already forced?

Reply:
Yes, it was "working" but the tasks was declared "parallel". So:

- the two tasks will successfully run concurrently **if scheduled to the same node**
- the two tasks may successfully run sequentially if scheduled to different nodes **and the second task does not time out**
- the second task _may_ time out if it is scheduled to another node and it takes too long time before it can execute/finish (I dont know how we implement time out)

**Consequentially:** If this is used in regression testing - the test may be **flaky** sometimes.

But this fixes so the two tasks always run in a sequence. For parallel tasks we should do as in the example provided in this commit. Unless we know that it will not time out.

Reply:
Ah, I see, great catch!